File: Programmers/[12987] 숫자 게임/Johoseong/12987.py
def solution(A, B):
    answer = 0
    A.sort()
    B.sort()
    
    while A:
        if B[-1] > A[-1]:
            answer += 1
            A.pop()
            B.pop()
        else:
            A.pop()

    return answer

print(solution([5,1,3,7], [2,2,6,8]))

File: Programmers/[12987] 숫자 게임/KingDonggyu/12987.js
function solution(A, B) {
  A.sort((a, b) => a - b);
  B.sort((a, b) => b - a);

  return B.reduce((acc, cur) => {
    while (A.length) {
      if (cur > A.pop()) {
        return acc + 1;
      }
    }
    return acc;
  }, 0);
}
